README.md

```diff
@@ -38,7 +38,24 @@ or downloading a file if it already exists with the same content locally or in t
 ## Installation
 
 An app bundle is provided for macOS High Sierra and higher and can be downloaded from the
-Releases tab. On other platforms, please download and install the Python package from PyPI:
+Releases tab. This app Bundle is also package as a Homebrew cask.
+
+On other platforms, you can download and install Maestral as a Python PyPI package, as a Docker package.
```

Comment:
```suggestion
On other platforms, you can download and install Maestral as a Python package from PyPI or as a Docker image from Docker Hub.
```

Its not a "PyPI package" but rather a Python package which happens to be hosted on PyPI...